files/en-us/web/api/webxr_device_api/fundamentals/index.html

```diff
@@ -19,23 +19,23 @@
 
-<p><span class="seoSummary">In this guide, we provide a fundamental overview of what WebXR is and how it works, as well as providing a foundation for what you'll need to know before you even begin to learn to develop augmented reality and virtual reality experiences for the web.</span></p>
+<p><span class="seoSummary">This guide provides an overview of what WebXR is and how it works, as well as the preliminary foundation needed to start developing augmented and virtual reality experiences for the web.</span></p>
 
 <h2 id="What_WebXR_is_and_isnt">What WebXR is and isn't</h2>
 
-<p>WebXR is an API for web content and apps to use to interface with mixed reality hardware such as VR headsets and glasses with integrated augmented reality features. This includes both managing the process of rendering the views needed to simulate the 3D experience and the ability to sense the movement of the headset or other motion sensing apparatus to provide the needed data to let you update the imagery shown to the user based on that movement.</p>
+<p>WebXR is an API for web content and apps to use to interface with mixed reality hardware such as VR headsets and glasses with integrated augmented reality features. This includes both managing the process of rendering the views needed to simulate the 3D experience and the ability to sense the movement of the headset (or other motion-sensing gear) and provide the needed data to update the imagery shown to the user.</p>
 
 <p>WebXR additionally provides support for accepting inputs from control devices such as handheld VR controllers or specialized mixed reality gamepads.</p>
 
-<p><em>WebXR is not a rendering technology, and provides no features for actually managing 3D data or rendering it to the display.</em> This is an important fact to keep in mind; while WebXR manages the timing, scheduling, and the various points of view relevant when drawing the scene, it does <em>not</em> actually know how to load and manage models, render and texture them, and so forth. That part is entirely up to you. Fortunately, WebGL and the various WebGL-based frameworks and libraries are available to make it much easier to deal with all of that.</p>
+<p><em>WebXR is not a rendering technology and does not provide features for managing 3D data or rendering it to the display.</em> This is an important fact to keep in mind. While WebXR manages the timing, scheduling, and the various points of view relevant when drawing the scene. It does <em>not</em> know how to load and manage models, nor how to render and texture them, and so forth. That part is entirely up to you. Fortunately, WebGL and the various WebGL-based frameworks and libraries are available to make it much easier to deal with all of that.</p>
```

Comment:
```suggestion
<p><em>WebXR is not a rendering technology and does not provide features for managing 3D data or rendering it to the display.</em> This is an important fact to keep in mind. While WebXR manages the timing, scheduling, and the various points of view relevant when drawing the scene, it does <em>not</em> know how to load and manage models, nor how to render and texture them, and so forth. That part is entirely up to you. Fortunately, WebGL and the various WebGL-based frameworks and libraries are available to make it much easier to deal with all of that.</p>
```
…otherwise that *While* part is gonna be a sentence fragment…

Reply:
I think that's what I wanted to do initially. I have no idea how this slipped through. Thanks for spotting this.